source/draw/RasterCanvas.ooc

```diff
@@ -18,7 +19,7 @@ import RasterImage
 RasterCanvas: abstract class extends Canvas {
 	_target: RasterImage
 	init: func (=_target) { super(this _target size) }
-	fill: override func { raise("RasterCanvas fill unimplemented!") }
+	fill: override func ~color (color: ColorRgba) { raise("RasterCanvas fill unimplemented!") }
```

Comment:
Because it would not compile without separating `~color` from `~pen`. I can remove it once the old version is gone.